README.md

```diff
@@ -34,6 +34,7 @@ The default mappings are as follows:
 - <kbd>K</kbd>, <kbd>J</kbd>: select prev or next tab
 - <kbd>r</kbd>: reload current tab
 - <kbd>R</kbd>: reload current tab without cache
+- <kbd>p</kbd>: toggle pin/unpin current tab
```

Comment:
please use another keys, because <kbd>p</kbd> is normally used for paste. Vim Vixen does not support paste URL currently, but I wanna reserve it.